.github/workflows/checks.yml

```diff
@@ -16,4 +16,32 @@ jobs:
           reporter: github-pr-review
           level: warning
           glob_pattern: "**/*.py"
-          filter_mode: "file"
\ No newline at end of file
+          filter_mode: "file"
+  frontend-checks:
+      runs-on: ubuntu-latest
+      defaults:
+        run:
+          working-directory: ./app
+      steps:
+        - uses: actions/checkout@v3
+
+        - uses: actions/setup-node@v3
+          with:
+            node-version: "19.x"
+
+        - name: Cache dependencies
+          uses: actions/cache@v3
+          with:
+            path: ~/.npm
+            key: npm-${{ hashFiles('**/package-lock.json') }}
+            restore-keys: |
+              npm-
+
+        - name: Install dependencies
+          run: npm ci --prefer-offline --no-audit
+
+        - name: Run linter
```

Comment:
This action doesn't appear to add review annotations to pull requests. I recommend using something like [reviewdog](https://github.com/reviewdog/reviewdog). I found these examples from reviewdog's github page: [eslint](https://github.com/reviewdog/action-eslint) and [prettier](https://github.com/EPMatt/reviewdog-action-prettier)